PowerPointLabs/PowerPointLabs/ActionFramework/TooltipsLab/Create/CreateTooltip/CreateTooltipActionHandler.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+
+using PowerPointLabs.ActionFramework.Common.Attribute;
+using PowerPointLabs.ActionFramework.Common.Extension;
+using PowerPointLabs.ActionFramework.Common.Interface;
+using PowerPointLabs.Models;
+using PowerPointLabs.TextCollection;
+using PowerPointLabs.TooltipsLab;
+using PowerPoint = Microsoft.Office.Interop.PowerPoint;
+
+namespace PowerPointLabs.ActionFramework.TooltipsLab
+{
+    [ExportActionRibbonId(TooltipsLabText.CreateTooltipTag)]
+    class CreateTooltipActionHandler : ActionHandler
+    {
+        protected override void ExecuteAction(string ribbonId)
+        {
+            this.StartNewUndoEntry();
+
+            PowerPointSlide currentSlide = this.GetCurrentSlide();
```

Comment:
Consider adding a guard clause that checks if `currentSlide `is null?

Reply:
The button will be disabled when there are no slides, but will add a guard clause!